src/main/java/com/conveyal/taui/models/AnalysisRequest.java

```diff
@@ -83,14 +86,15 @@ public AnalysisTask populateTask (AnalysisTask task, Project project) {
             modifications = modificationsForProject(project.accessGroup, projectId, variantIndex);
         }
 
-        // No idea how long this operation takes or if it is actually necessary
+        // Generate a checksum of all the modifications
         CRC32 crc = new CRC32();
-        crc.update(modifications.stream().map(Modification::toString).collect(Collectors.joining("-")).getBytes());
+        modifications.stream().map(JsonUtilities::objectToJsonBytes).forEach(crc::update);
```

Comment:
Is there a reason to do the list iteration in our code? It seems like we could just serialize the whole list object to JSON and CRC that.

Reply:
for example: `crc.update(JsonUtilities.objectToJsonBytes(modificationsForProject(project.accessGroup, projectId, variantIndex)));`

Reply:
Good call 👍